lnrpc/lightning.proto

```diff
@@ -591,6 +591,19 @@ service Lightning {
     */
     rpc ListAliases (ListAliasesRequest) returns (ListAliasesResponse);
 
+    /*
+    AddAliases creates a set of new alias mappings. The list of successfully
```

Comment:
Argh, sorry... @ffranr and I talked about this offline but forgot to forward the thoughts to you... We should put this into the `routerrpc` package, since for now it's only really relevant there (and the main package is too overloaded already). And add an `X` prefix as we did with some mission control RPCs to mark them as experimental.

My proposal:

```
    /*
    XAddLocalChanAliases is an experimental API that creates a set of new
    channel SCID alias mappings. The list of successfully created mappings is
    returned. This is only a locally stored alias, and will not be communicated
    to the channel peer via any message. Therefore, routing over such an alias
    will only work of the peer also calls this same RPC on their end.
    */
    rpc XAddLocalChanAliases (AddAliasesRequest) returns (AddAliasesResponse);

    /*
    XDeleteLocalChanAliases is an experimental API that deletes a set of new
    alias mappings. The list of successfully deleted mappings is returned. The
    deletion will not be communicated to the channel peer via any message.
    */
    rpc XDeleteLocalChanAliases (DeleteAliasesRequest)
        returns (DeleteAliasesResponse);
```

Then later, when we actually add the p2p communication, we can add a _new_ RPC to the `peersrpc` package and mark these as deprecated.

Also, the commit message should be updated to mention `routerrpc`.